protected/humhub/widgets/Pjax.php

```diff
@@ -11,6 +11,8 @@ class Pjax extends \yii\widgets\Pjax
     public function init()
     {
         $this->options = ArrayHelper::merge($this->options, [
+            'class' => (array) ArrayHelper::getValue($this->options, 'class', [])
+        ], [
```

Comment:
We should use a suggested code from here https://github.com/humhub/humhub/pull/6713#issuecomment-1855389596:

```php
$class = (array)ArrayHelper::getValue($this->options, 'class', []);
$class[] = 'exclude-from-pjax-client';
$this->options['class'] = $class;
```

Reply:
@yurabakhtin why? both are doing the same thing

Reply:
@gevorgmansuryan if you'd read the conversation above, you'd see that 
a) your code creates duplicate entries (https://github.com/humhub/humhub/pull/6713#issuecomment-1855619202)
b) does unnecessary array creation and merges and hence isn't very efficient (https://github.com/humhub/humhub/pull/6713#issuecomment-1855389596)

Of course only if you replace the entire code (Lines 13-17).

Personal remark: Sometimes its worth double-checking if someone else spotted a mistake. I already pointed it out a few times before you actually looked at it. Then you made a quick-shot-fix without due diligence. And now again you just assume your code is correct. However, it's not ... Just a gentle reminder.

Reply:
@martin-rueegg Thanks for remarks :)